app/controllers/devise/two_factor_authentication_controller.rb

```diff
@@ -34,8 +34,6 @@ def reauthn_param
 
     def handle_valid_delivery_method(method)
       send_user_otp(method)
-      resent_message = t("notices.send_code.#{method}")
-      flash[:success] = resent_message if session[:code_sent].present?
```

Comment:
so are we not showing this message anymore? If not, do we still want to set the session var below?